_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -275,12 +283,53 @@ class HealthCareApp extends React.Component {
           external: this.getExternalData(this.state.applicationData, statePath)
         });
       });
+    }
 
-      if (this.props.location.pathname === '/review-and-submit') {
-        buttons = (<SubmitButton onButtonClick={this.handleSubmit}/>);
-      } else {
-        buttons = (<ContinueButton onButtonClick={this.handleContinue}/>);
-      }
+    // Check which section the user is on and render the correct ProgressButtons.
+    const lastSectionText = (this.getUrl('back')) ? this.getUrl('back').split('/').slice(-1)[0].replace('-', ' ') : '';
+    const nextSectionText = (this.getUrl('next')) ? this.getUrl('next').split('/').slice(-1)[0].replace('-', ' ') : '';
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={this.handleBack}
+          buttonText={`Back to ${lastSectionText}`}
+          buttonClass={'usa-button-primary-alt'}/>
+    );
+
+    const nextButton = (
+      <ProgressButton
+          onButtonClick={this.handleContinue}
+          buttonText={`Continue to ${nextSectionText}`}
+          buttonClass={'usa-button-primary'}/>
+    );
+
+    const submitButton = (
+      <ProgressButton
+          onButtonClick={this.handleSubmit}
+          buttonText={'Submit'}
+          buttonClass={'usa-button-primary'}/>
+    );
+
+    if (this.props.location.pathname === '/review-and-submit') {
+      buttons = (
+        <div>
+          {backButton}
+          {submitButton}
+        </div>
+      );
+    } else if (this.props.location.pathname === '/introduction') {
+      buttons = (
+        <div>
+          {nextButton}
+        </div>
+      );
+    } else {
+      buttons = (
+        <div>
+          {backButton}
+          {nextButton}
+        </div>
+      );
```

Comment:
This is all great! I'm wondering if we want to extract this button logic into another file? Doesn't have to be in this PR, just wondering if that's a good idea eventually. Might help declutter `HealthCareApp` and help contain all of this button logic into it's own thing?